storybook/src/elements/actions/actions.stories.js

```diff
@@ -0,0 +1,209 @@
+import '@nuxeo/nuxeo-ui-elements/actions/nuxeo-add-to-collection-button';
+import '@nuxeo/nuxeo-ui-elements/actions/nuxeo-favorites-toggle-button';
+import '@nuxeo/nuxeo-ui-elements/actions/nuxeo-delete-blob-button';
+import '@nuxeo/nuxeo-ui-elements/actions/nuxeo-delete-document-button';
+import '@nuxeo/nuxeo-ui-elements/actions/nuxeo-download-button';
+import '@nuxeo/nuxeo-ui-elements/actions/nuxeo-export-button';
+import '@nuxeo/nuxeo-ui-elements/actions/nuxeo-link-button';
+import '@nuxeo/nuxeo-ui-elements/actions/nuxeo-lock-toggle-button';
+import '@nuxeo/nuxeo-ui-elements/actions/nuxeo-move-documents-down-button';
+import '@nuxeo/nuxeo-ui-elements/actions/nuxeo-move-documents-up-button';
+import '@nuxeo/nuxeo-ui-elements/actions/nuxeo-notifications-toggle-button';
+import '@nuxeo/nuxeo-ui-elements/actions/nuxeo-share-button.js';
+import '@nuxeo/nuxeo-ui-elements/actions/nuxeo-untrash-document-button.js';
+
+import { html } from 'lit-html';
+import { storiesOf } from '@storybook/polymer';
+import { color } from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
+import '@nuxeo/nuxeo-ui-elements/nuxeo-icons';
```

Comment:
Prettier does not order imports? This one should be "upstairs", correct?

Reply:
Probably Prettier and ESLint is not setup for Storybook? I will order them manually for now and look into it in the next sprint.